### DISCUSSION TITLE
Variables: Save dashboard variables per user

### DISCUSSION THREAD

**Discussion by yoyo421** [grafana/grafana#44523] [Dashboards, Panels & Visualizations]
**What would you like to be added:**
Add a fallback if a dashboard variable is not set in the URL query, check what the user set the variable last time

**Why is this needed:**
If I have a couple of variables, that uses a lot of options, every time I open the dashboard I need to start selecting the variables again from a very long list of options, and it gets really frustrating really fast, because I'm not the owner of the dashboard I cannot save my changes and keep it for the next time (other people working on this dashboard but interested in a different set of selections).

Keeping track of the URL every time I change a variable will not help because I open the dashboard from Grafana where the default URL is without the variables.

**Comment by JaimeAvalos** [comment]
This would be highly useful. I am working with a large list of users and variables but each of them is only interested in one variable. It would be very handy that the dashboard with last session's variables.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82999. Feel free to continue the discussion around this there. Thank you!
